data/Santini_2012/metadata.yml

```diff
@@ -0,0 +1,201 @@
+    at Giralia Bay is inundated by the tide twice daily. ^p At each site (three at
+    NZ and one in WA), we assessed the porewater salinity. We collected water samples
+    from 30 cm depth using a suction device (McKee et al. 1988) and analysed them
+    with a handheld refractometer (W/ATC 300011, SPER Scientific Scottsdale, USA).
+  original_file: AusTRAITS_Santini (1).xls in raw data folder
+  notes: none
+locations:
+  Giralia Bay, Exmouth Gulf, Western Australia:
+    latitude (deg): -22.4
+    longitude (deg): 114.3
+    precipitation, MAP (mm): 300
+    temperature, MAT (C): 25
+contexts: .na
+traits:
+- var_in: Wood density (g/cm3)
+  unit_in: g/cm3
+  trait_name: wood_density
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: '6'
+  methods: All our measurements were made in the 3-4 outermost layers of wood (Fig.
+    1a). Wood density was measured for each tree from 1 cm x 1 cm x 2 cm rectangular
+    prisms cut from the outer layers of the wood. Wood density was calculated as the
+    ratio of the oven-dry mass of the wood sample divided by the mass of water displaced
+    by its fresh (green) volume (Chave et al. 2006). Using oven-dried (60 °C) and
+    sanded wood discs (Fig. 1a, b), the number of growth layers, defined as ontogenetic
+    units of phloem and xylem, was counted using an Olympus SZH10 microscope (Tokyo,
+    Japan). A growth layer was counted if it crossed a line drawn from pith to bark
+    along the maximum radius of the stem disc (Schmitz et al. 2007). The proportion
+    of phloem per growth layer was calculated as the width of the phloem band divided
+    by the total width of the growth layer, including both xylem and phloem, measured
+    along the radius of the stem disc. Given it can be difficult to resolve small
+    vessels from sanded wood discs, ~1 cm3 cubes were cut from the discs to make transverse
+    microsections (Fig. 1c) that were double stained with Safranin-Alcian blue. We
+    photographed the sections with an Olympus BX61 microscope (Tokyo, Japan). From
+    the photographs we measured 100-200 xylem vessels and 100-200 fibres for each
+    stem. Xylem and phloem area was measured per layer. The number of vessels was
+    counted and the perpendicular axes of the vessel lumina were measured (Fig. 1c).
+    All the measurements were done using Image Pro Plus version 5.0.1 (MediaCybernetics,
+    Maryland, USA). We calculated the xylem vessel diameter (D) following the ellipse
+    equation of Lewis (1992) (Eq. 1), and the total vessel lumen area (XA) accounting
+    for vessel density (VD) and vessel area (VA). Fibre wall thickness was calculated
+    as the radial distance from fibre lumen 1 to fibre lumen 2 divided by two.
+- var_in: '%Phloem Ring'
+  unit_in: '%'
+  trait_name: stem_vessel_lumen_fraction
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
```

Comment:
I think it means the measurements were made across a thickness that represented 3-4 wood layers. Since it is a  '%' of the stem and is for 1 stem, I think this is appropriate